native/ssz_nif/src/lib.rs

```diff
@@ -21,7 +21,6 @@ mod atoms {
 }
 
 const SCHEMA_PREFIX_SIZE: usize = "Elixir.Types.".len();
-const ELIXIR_PREFIX_SIZE: usize = "Elixir.".len();
 
 #[rustler::nif]
 fn to_ssz_rs<'env>(env: Env<'env>, map: Term, schema: Atom, config: Atom) -> NifResult<Term<'env>> {
```

Comment:
we probably should rename `config` to `preset` everywhere in the `ssz_nif` module to avoid confusion.

Reply:
#604